include/seastar/util/noncopyable_function.hh

```diff
@@ -57,6 +57,10 @@ private:
         new (&to->_storage.indirect) void_ptr(from->_storage.indirect);
     }
 
+    static void ref_move(noncopyable_function_base* from, noncopyable_function_base* to) noexcept {
```

Comment:
> util: noncopyable_function: add reference_wrapper constructor
> So a noncopyable_function can be used by reference
rather than being copied.

Will it really be copied? the reference_wrapper object will be copied, but that's just a pointer.

Reply:
We need a way to construct a noncopyable_function from a reference_wrapper

Reply:
This compiles: https://godbolt.org/z/hjv6n5zE6